### ISSUE TITLE
Reproducibility issue

### ISSUE THREAD

**Issue by Dnekhaev** [facebookresearch/deit#40]
Thanks for great work.
I tried to reproduce learning of tiny, small and base models using your code. The only parameter I had to change is batch size due to resource constraints. I successfully reproduced results for small and tiny networks but have big accuracy drop for base.

I got:
Tiny - Acc@1 72.172 Acc@5 91.188 loss 1.222 Max accuracy: 72.31% (batch size 256 per gpu, 8 gpus)
Small - Acc@1 79.786 Acc@5 95.008 loss 0.880 Max accuracy: 79.84% (batch size 144 per gpu, 8 gpus)
Base - Acc@1 78.568 Acc@5 93.966 loss 1.048 Max accuracy: 78.78% (batch size 60 per gpu, 8 gpus)

<img width="427" alt="Deit-BS" src="https://user-images.githubusercontent.com/22977714/104344085-c0a66c00-550d-11eb-8d9e-a74a0e54c0f1.png">

On learning curve I see that base model starts overfitting. Base model has higher test loss and lower train loss than small model. Could be smaller batch size the reason of such big drop of accuracy (-3 % Acc@1)? Did you reproduce results of base model with this code and training parameters?

**Comment by fmassa** [comment]
Hi,

Thanks for trying out DeiT.

Here are the results I got by running the base model on commit b06fcba6bf98740dcce3d0627856610f378afd0d with default parameters, on 16 GPUs:

```
python run_with_submitit.py --model deit_base_patch16_224
```

```
* Acc@1 81.660 Acc@5 95.512 loss 0.838
* Max accuracy: 81.73%
```
Which matches the reported results (within std).

To facilitate reproduction, here is the training logs for the run just above: https://gist.github.com/fmassa/0497ed4e382d5de19b98bb7168815610


Given that you have changed the total batch size for your experiments, I believe you might need to adapt the other hyperparameters (learning rate etc) to get comparable results.
Another thing that you could try would be to do gradient accumulation in order to run the model with comparable batch sizes on 8 GPUs.

Given that we are able to reproduce the results with the provided code, I'm closing this issue but let us know if you have further questions.

**Comment by lcmeng** [comment]
@fmassa or @Dnekhaev can you please share the training log for DeiT_tiny_patch16_224 without distillation, the one that achieves 72.2% top-1 accuracy? TIA.
